src/WinUI/ArcGIS.WinUI.Viewer/SamplePage.xaml.cs

```diff
@@ -37,6 +39,8 @@ public SamplePage()
             // Load and show the sample.
             SampleContainer.Content = SampleManager.Current.SampleToControl(SampleManager.Current.SelectedSample);
 
+            (SampleContainer.Content as FrameworkElement).Unloaded += SamplePage_Unloaded;
```

Comment:
Event is already being used for ContentPage.Unloaded, and you're using the same eventhandler. Is this line even needed? The Unloaded handler should already be called